### ISSUE TITLE
Links to examples seem broken or they aren't showing?

### ISSUE THREAD

**Issue by joshgoebel** [scotch-io/10-percent-too-dull-for-my-tastes#5]

**Comment by asep19** [comment]
Yeah the link is broken, it should be link to   [https://cdn.scotch.io/2/xwxZZIowT5q6jVLUAwCc_javascript.gif](https://cdn.scotch.io/2/xwxZZIowT5q6jVLUAwCc_javascript.gif)
 
scotch.io website doesn't exist anymore, because:
 
> [Scotch.io is joining DigitalOcean](https://www.digitalocean.com/blog/scotch-io-is-joining-digitalocean)



You can see the link to the image from Raw README.MD.
